apps/site/test/site_web/controllers/project_controller_test.exs

```diff
@@ -7,6 +7,13 @@ defmodule SiteWeb.ProjectControllerTest do
   alias Plug.Conn
 
   describe "project_updates" do
+    test "successfully renders the project update list", %{conn: conn} do
+      path = project_updates_path(conn, :project_updates, "project-name")
+
+      conn = get(conn, path)
+      assert conn.status == 200
```

Comment:
Can we go a bit further here? Something like

```elixir
assert html_response(conn, 200) =~ "some string that should be on the page"
```

Reply:
I ensured that `CMS.Repo.get_page/2` is natively called via this test, which is all the fix needed. It would have otherwise issued a 500 response for `conn.status` prior to the fix.

Reply:
Right, I understand this was the minimum required to expose the bug at hand. I was wondering if we couldn't get some extra value out of this test, while we're here, for a small additional effort.